src/classes/RD2_OpportunityMatcher.cls

```diff
@@ -132,6 +132,22 @@ public inherited sharing class RD2_OpportunityMatcher {
         return this;
     }
 
+    /***
+     * @description Filters open Opportunities regardless of the Close Date and
+     * open/closed Opportunities with Close Date greater or equal to the current date.
+     * Opportunities marked as Closed Lost in data migration are ignored.
+     * @param opps Recurring Donation Opportunities
+     * @return RD2_OpportunityMatcher This Opportunity matcher instance
+     */
+    public RD2_OpportunityMatcher filterCurrentOrOpen(List<Opportunity> opps) {
```

Comment:
I'm not particularly clear on the use of the word "filter" through this code. I generally take that to mean "filter out" (exclude) records. However, I think that the intent of this is to choose which recors to include. If that's the case, I think it might be a good idea to replace all uses of `filter` to `include` or `exclude` depending on the intent of the method.

Reply:
Thanks for the clarification. Personally I do find it confusing. For example, the test method `shouldFilterAllOppsWhenCloseDateYearMatchesCurrentYear` reads to me as "it should filter out all close date matches the current year". Same with this method. I read that as "filter out current or open opps". I think of a filter as something that removes items, so anything listed after the word filter (to me) is filtered out. 

For this method, I think that either `includeCurrentOrOpenOpps` or `filterToCurrentOrOpen` would work better (the latter adding "to" so it's explicitly stating the inclusion vs. the default exclusion). It's minor, but I think long term it'll be easier to read back in 6 months and know exactly what's happening.

Reply:
The change has been done, thanks @force2b  for the feedback!